styles/index.css

```diff
@@ -290,3 +290,153 @@ button .before-icon{
     color:var(--pink8);
     padding-left:2rem;
 }
+
+.nav-toggle{
+    display: none;
+}
+
+.nav-toggle-label{
+    display: none;
+}
+
+/* responsive navbar */
+@media only screen and (max-width:1000px){
+
+    header {
+        display: grid;
+        grid-template-columns: 1fr;
+        width: 100%;
+    }
+
+    .nav-toggle {
+        position: absolute !important;
+        top: 50% !important;
+        /* left: 1px; */
+        background-color: var(--dark-red1);
+    }
+
+    .search-head{
+        display: none;
+    }
+
+    .actions-head .actions-icon{
+        display: none;
+    }
+
+    .logo-head{
+      display: none;
+    }
+
+    .nav-toggle-label {
+        position: absolute;
+        top: 0;
+        left: 0;
+        margin-left: 0.1em;
+        height: 100%;
+        display: flex;
+        align-items: center;
+        min-width: 4rem;
+    }
+
+    .nav-toggle-label span,
+    .nav-toggle-label span::before,
+    .nav-toggle-label span::after {
+        display: block;
+        background: white;
+        height: 2px;
+        width: 2em;
+        border-radius: 2px;
+        position: relative;
+    }
+    
+    .nav-toggle-label span::before,
+    .nav-toggle-label span::after {
+        content: '';
+        position: absolute;
+    }
+    
+    .nav-toggle-label span::before {
+        bottom: 7px;
+    }
+    
+    .nav-toggle-label span::after {
+        top: 7px;
+    }
+
+    .catLinks-head {
+        position: absolute;
+        text-align: left;
+        top: 100%;
+        left: 0;
+        background: rgba(224,203,143,0.2);
```

Comment:
i was trying to change the transparency of a certain color so that i can use it for toggle nav. will update it later. i will do all these refactors later. thanks!